### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by joshuarli**

### CODE REVIEW

File: src/docs/performance-monitoring/performance/event-detail.mdx [hunk 1]
```diff
@@ -82,7 +82,7 @@ import sentry_sdk
 
 #  ...
 
-columns = ["first_column", "second_column", "third_columns", "fourth_column", "fifth_column", "sixth_colummn"]
+columns = ["first_column", "second_column", "third_columns", "fourth_column", "fifth_column", "sixth_column"]
```
Comment (rhcarvalho) [inline src/docs/performance-monitoring/performance/event-detail.mdx:85]: @joshuarli for a future pass:

```diff
-third_columns
+third_column
```

:)

I've done a similar mass spellchecking in Django docs in a hackday some years ago, good stuff!

File: src/docs/platforms/php/laravel.mdx [hunk 2]
```diff
@@ -215,8 +215,7 @@ $ php artisan sentry:test
 ### Local development
 
 When Sentry is installed in your application it will also be active when you are developing.
-
-If you don't want errors to be sent to Sentry when you are developing set the DSN value to `null`.
+If you want errors to be sent to Sentry when you are developing set the DSN value to `null`.
```
Comment (rhcarvalho) [inline src/docs/platforms/php/laravel.mdx:218]: Oooops, I think this was a bad change?

Comment (untitaker) [reply src/docs/platforms/php/laravel.mdx:218]: https://github.com/getsentry/sentry-docs/pull/1974

**Comment by untitaker** [comment]
@joshuarli would not have expected anybody to merge into master this week other than to hotfix stuff

**Pull request merged**